search-provider/eks-discovery-feed-provider.c

```diff
@@ -120,20 +120,44 @@ object_path_from_app_id (const gchar *application_id)
 }
 
 static gboolean
-ensure_app_proxy (EksDiscoveryFeedDatabaseContentProvider *self)
+ensure_content_app_proxy (EksDiscoveryFeedDatabaseContentProvider *self)
+{
+  if (self->content_app_proxy != NULL)
+    return TRUE;
+
+  g_autofree gchar *object_path = object_path_from_app_id (self->application_id);
+  GError *error = NULL;
+  self->content_app_proxy = eks_discovery_feed_content_proxy_new_for_bus_sync (G_BUS_TYPE_SESSION,
+                                                                               G_DBUS_PROXY_FLAGS_DO_NOT_AUTO_START_AT_CONSTRUCTION |
+                                                                               G_DBUS_PROXY_FLAGS_DO_NOT_LOAD_PROPERTIES,
+                                                                               self->application_id,
+                                                                               object_path,
+                                                                               NULL,
+                                                                               &error);
+  if (error != NULL)
+    {
+      g_warning ("Error initializing dbus proxy: %s\n", error->message);
```

Comment:
`g_warning()` already puts a newline at the end (also a problem for other `g_warning()` calls in the file probably -- please fix the other occurrences too)